erigon-lib/commitment/commitment.go

```diff
@@ -1048,6 +1061,10 @@ func (t *Updates) TouchPlainKey(key string, val []byte, fn func(c *KeyUpdate, va
 			}
 			t.keys[key] = struct{}{}
 		}
+	case ModeInMemory:
+		if _, ok := t.keys[key]; !ok {
+			t.keys[key] = struct{}{}
```

Comment:
what if insert hashed key to list now and sort in bg?

Reply:
What is bg?

Reply:
If you hash the key now you will lose the benefits of parallelization, since each insert happens serially.

Reply:
bg=background

Reply:
at least put it into some hashing queue where workers do pick and then just use sort.Sort over finished list of pairs <hashed_key, plain_key>